include/glow/Backends/BackendUtils.h

```diff
@@ -24,15 +24,15 @@ namespace runtime {
 /// Contains information for initialization and handling of symbol at runtime.
 struct RuntimeSymbolInfo {
   /// The size in bytes.
-  size_t size;
+  size_t size{0};
   /// Offset in bytes from the base address.
-  size_t offset;
+  size_t offset{0};
   /// Type of symbol.
   Type type;
   /// Is the symbol an input for the function.
-  bool input;
+  bool input{false};
```

Comment:
Question: I wonder if `true` is a more natural value. E.g. if you'd have a logic somewhere that says "copy all symbols that are inputs to devices and then copy all symbols that are outputs from devices", the `true` value would result in copying all the required symbol and may be more. But with `false` value, nothing would be copied in either direction.

Reply:
I thought about that too, ultimately my thinking was default to false and if a symbol needs to be copied it should be set to true. I'm good either way though.

Reply:
> if a symbol needs to be copied it should be set to true. 

Do we have the code that sets it to true if needed? Is this code invoked on every path leading to "execute" or the like?

Reply:
That's a fair point. It is in the runtime but not in the executionengine. I'll change it to true.